kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -413,7 +413,8 @@ private void translateSymbol(Map<Att.Key, Boolean> attributes, SetMultimap<KLabe
         sb.append(") : ");
         convert(prod.sort(), prod, sb);
         sb.append(" ");
-        Att koreAtt = addKoreAttributes(prod, functionRules, overloads);
+        Att koreAtt = prod.att();//addKoreAttributes(prod, functionRules, overloads);
```

Comment:
I'm having issues here, probably due to the Module transformer, as even though I can add the Attributes to the production, they doesn't appear updated here.